.github/workflows/zulip-tests.yml

```diff
@@ -59,3 +57,29 @@ jobs:
 
       - name: Codecov
         uses: codecov/codecov-action@v1.2.1
+
+  test-windows:
+    runs-on: windows-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.7, 3.8, 3.9]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install dependencies
+        run: python tools\provision --force
+
+      - name: Running Test-Suite
+        run: |
+          zulip-api-py3-venv\Scripts\Activate.ps1
+          tools\test-main.ps1
```

Comment:
Except for these, this set of jobs are the same as the linux tests, so can this be simpler as part of the same matrix?

This is a new test script, so do we plan to document it? Do we plan to support only WSL or native windows too - this script is only necessary for the latter?

Reply:
While running tests for python 3.6, installing `cryptography` gave an error on Windows (It required linking to OpenSSL's C library but failed since pip in 3.6 is quite old). So, I skipped 3.6 on Windows.

The test script is only for native Windows. One can run already present bash scripts on WSL.

Reply:
OK -- maybe add a comment that this is why these are duplicated?  Or perhaps we should look for a better way to skip testing 3.6 on Windows in the matrix feature.